### PR TITLE
Bump version 0.11.0

### PR DESCRIPTION
**Pull request by mkundu1**

### CODE REVIEW

**Review by dnwillia-work — APPROVED** [review]

File: src/ansys/fluent/core/_version.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 """
 
 # major, minor, patch
-version_info = 0, 11, "dev4"
+version_info = 0, 11, 0
```
Comment (dnwillia-work) [inline src/ansys/fluent/core/_version.py:9]: Should the dev version not be incremented to the next minor release? i.e.
```suggestion
version_info = 0, 12, "dev0"
```

Comment (mkundu1) [reply src/ansys/fluent/core/_version.py:9]: Yes, that's right. I won't merge this PR, will update the dev version from a different branch.

Comment (dnwillia-work) [reply src/ansys/fluent/core/_version.py:9]: OK, that's what I thought, thanks!  I had it in my head that 1. We create a release branch for 0.11.0, 2. Increment the main branch to the next version 3. Optionally delete the 0.11.0 release branch but tag the commit.

**Comment by mkundu1** [comment]
Released.

**Pull request closed without merging**